File: sdk/scala-play-server/app/api/ProductConversionApiImpl.scala
package api

import model.InlineResponse200
import model.InlineResponse400

/**
  * Provides a default implementation for [[ProductConversionApi]].
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2020-03-27T18:11:16.680-04:00[America/New_York]")
class ProductConversionApiImpl extends ProductConversionApi {
  /**
    * @inheritdoc
    */
  override def convertCode(upc Pipe ean Pipe asin: String): InlineResponse200 = {
    // TODO: Implement better logic

    InlineResponse200(None, None, None, None)
  }
}

File: sdk/java-vertx-web/src/main/java/org/openapitools/vertxweb/server/api/ProductConversionApiImpl.java
package org.openapitools.vertxweb.server.api;

import org.openapitools.vertxweb.server.model.InlineResponse200;
import org.openapitools.vertxweb.server.model.InlineResponse400;

import org.openapitools.vertxweb.server.ApiResponse;
import org.openapitools.vertxweb.server.ApiException;

import io.reactivex.Single;

import java.util.List;
import java.util.Map;

// Implement this class

public class ProductConversionApiImpl implements ProductConversionApi {
    public Single<ApiResponse<InlineResponse200>> convertCode(String upcEanAsin) {
        return Single.error(new ApiException("Not Implemented").setStatusCode(501));
    }

}
